packages/async-flow/src/replay-membrane.js

```diff
@@ -154,14 +154,9 @@ export const makeReplayMembrane = ({
     const passStyle = passStyleOf(h);
```

Comment:
I love that we can now stop tolerating host promises and instead reject them with errors!

However, this leaves the doc-comment above and the function name `tolerate*` stale. Please revise to be consistent either with consistent rejection, or at least as the function which decides whether to tolerate or to reject. I don't have a name to suggest.

Reply:
Ok to fully remove `tolerateHostPromiseToVow`? I was trying to be surgical because I didn't know if you needed it to be there.